### PR TITLE
Use 'shouldAbortTest' to check if testing shall continue or end early

### PR DESCRIPTION
**Pull request by tomasz-lisowski**
Seems like 'failOrAbort' was not used in the check/require implementations.

### CODE REVIEW

**Comment by jasmcaus** [comment]
For sanity reasons, could you please change all instances of `if (...)` to `if(...)` (no space between the `if`s

**Comment by tomasz-lisowski** [comment]
On that note actually, I wasn't sure what formatting style you used. Could you add a `.clang-format` definition file or something along those lines to make the style explicit? In the meantime I will remove the spaces.

**Comment by jasmcaus** [comment]
I've never used any external formatting tools ever, but maybe I should. My formatting is very simple (and universal), so not that hard to follow.
Thanks for your contribution! Will merge.
